WebDevStudios/nextjs-wordpress-starter
# Fix/media text gutenberg fixes

Closes N/A

### Link

N/A

### Description

* Modify the MediaText component to allow arbitrary children on the text side.
  * This needed to be done because the Gutenberg MediaText block allows any blocks inside the Text area

### Screenshot

N/A

### Verification

How will a stakeholder test this?

1. Checkout the branch
2. Run storybook
3. Verify that the MediaText block can accept
